starter-code/src/functions-and-arrays.js

```diff
@@ -59,6 +138,16 @@ var wordsFind = [
   'truth',
   'disobedience'
 ];
+function doesWordExist (array, word) {
+  var wordExists = false
+  for (var i = 0; i<array.length; i++) {
+    if (word == array[i]) {
+      wordExists = true
+    } 
+  }
+  return wordExists
+}
+console.log(doesWordExist(wordsFind,"disobedienc"))
 
```

Comment:
Nice job Ana! Maybe as a bonus try to solve this one also with a higher function like filter?
Kido